### DISCUSSION TITLE
Company invoice

### DISCUSSION THREAD

**Discussion by juchom** [statiqdev/Discussions#146] [Q&A] [answered]
Hello @daveaglick,

After some testing in my company, we really like how Statiq is designed and we're going to purchase 3 licenses.

Is there a way to have an invoice with my company information after the stripe payment ?

Thanks again for this great tool 👍

**Comment by daveaglick** [comment, marked as answer]
That's awesome! I have some boilerplate invoices that I've sent to other licensees so that's not a problem and I'm happy to do it. Let me know if you also need a quote beforehand to support a PO process (I know at my own employer I have to obtain a quote, create a PO, and get that approved before I can purchase anything).

Also note that your timing is good and you may not even need a license - see the discussion at https://github.com/orgs/statiqdev/discussions/147. A license change has been in the works and I'm finally getting around to switching everything over. Small businesses with less than $100,000 total revenue and 10 employees are no longer required to purchase licenses. This was kind of always the intent, but it wasn't explicit and I was individually letting folks know, so now it'll be spelled out explicitly. Not sure if that changes your situation or not.

In any case, I'll keep an eye out for a license purchase if you end up making one and send over an invoice. Feel free to contact me directly at dave[at]daveaglick.com if you've got any specific questions related to purchasing or licensing.

Glad you're enjoying the tool (and I'll get back to your other discussions soon - been out of town and out of pocket a lot recently).

**Reply by juchom** [reply]
We're under the limit, but we will take it anyway !

We really like the job you have done, we can afford it, so we are happy to reward your work.

Thanks again for this, I'll do the purchase and send you an email!
